atlasdb-client/src/main/java/com/palantir/atlasdb/transaction/impl/TransactionConstants.java

```diff
@@ -58,6 +59,7 @@ public static long getTimestampForValue(byte[] encodedTimestamp) {
     public static final TableMetadata TRANSACTION_TABLE_METADATA = TableMetadata.internal()
             .singleRowComponent("write_ts", ValueType.VAR_LONG)
             .singleNamedColumn(COMMIT_TS_COLUMN_STRING, "commit_ts", ValueType.VAR_LONG)
+            .nameLogSafety(LogSafety.SAFE)
             .build();
```

Comment:
uh can we add SweepStrategy.NOTHING here for consistency?

Reply:
done